configs/kubernetes-rules-configmap.yaml

```diff
@@ -846,3 +846,79 @@ data:
           branch: release-1.12
         - repository: client-go
           branch: release-9.0
+    - destination: sample-cli-plugin
+      library: true
+      branches:
+      - source:
+          branch: master
+          dir: staging/src/k8s.io/sample-cli-plugin
+        name: master
+        dependencies:
+        - repository: api
+          branch: master
+        - repository: apimachinery
+          branch: master
+        - repository: cli-runtime
+          branch: master
+        - repository: client-go
+          branch: master
+    - destination: cli-runtime
+      library: true
+      branches:
+      - source:
+          branch: master
+          dir: staging/src/k8s.io/cli-runtime
+        name: master
+        dependencies:
+        - repository: api
+          branch: master
+        - repository: apimachinery
+          branch: master
+        - repository: client-go
+          branch: master
+    - destination: kube-proxy
+      library: true
+      branches:
+      - source:
+          branch: master
+          dir: staging/src/k8s.io/kube-proxy
+        name: master
+        dependencies:
+        - repository: apimachinery
+          branch: master
+    - destination: kubelet
+      library: true
+      branches:
+      - source:
+          branch: master
+          dir: staging/src/k8s.io/kubelet
+        name: master
+        dependencies:
+        - repository: apimachinery
+          branch: master
+        - repository: api
```

Comment:
client-go is missing

Reply:
nvmd. With the config apis only right, porbably it is complete

Reply:
y i checked with GoDeps